src/py21cmfast/src/SpinTemperatureBox.c

```diff
@@ -1,14 +1,15 @@
-
+#include "JSC.h"
 // Re-write of find_HII_bubbles.c for being accessible within the MCMC
+// Note for JSC: Anything that begins with Radio or Hawking are my modules
```

Comment:
maybe calling the module "JSC" isn't the best idea for the long-term. Could we call it something more meaningful?

Reply:
Hi Steven, the JSC signatures in previous PR has now been removed

Reply:
the JSC signatures in previous PR has now been removed

Reply:
and the JSC.h has also been renamed as RadioExcess.h